test-case/check-kmod-load-unload-after-playback.sh

```diff
@@ -57,24 +57,28 @@ func_lib_check_sudo
 # so when load the test-case in current script
 # the test-case will write the log to the store folder LOG_ROOT
 # which is current script log folder
-export LOG_ROOT=$LOG_ROOT
+log_root=$LOG_ROOT
 
 if [ ${OPT_VALUE_lst['p']} -eq 1 ];then
     func_lib_disable_pulseaudio
 fi
 
+export LOG_ROOT=$log_root/round-0
 $(dirname ${BASH_SOURCE[0]})/check-playback.sh -l 1 -t $tplg -d $pb_duration
 ret=$?
+export LOG_ROOT=$log_root
```

Comment:
don't need to export here

Reply:
turn back the `LOG_ROOT` to current log folder

Reply:
Do we need to turn back here?

Reply:
yes, this value common use at [exit function](https://github.com/thesofproject/sof-test/blob/master/case-lib/hijack.sh#L19)   
we define it for when your current case need to write some file will write to `LOG_ROOT`  
here this change, because have sub-case